### ISSUE TITLE
Volume Calculation Module

### ISSUE THREAD

**Issue by benjaminwinger** [UWARG/computer-vision#9]

**Comment by dmurds** [comment]
Space carving looks like an interesting theorem... link here, 
http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.163.250 , http://stackoverflow.com/questions/7161209/determine-the-volume-of-an-object-from-pictures is a stack overflow question on the topic

**Comment by dmurds** [comment]
Look at calculating volume by point cloud: changes in pixels can be used to calculate changes in height, then perform shitty integration to determine more accurate volume.

**Comment by dmurds** [comment]
http://stackoverflow.com/questions/20314556/generating-point-cloud-from-many-2d-images
Point cloud question; top comment has some good papers and stuff.

**Comment by dmurds** [comment]
Stuff we're discovered so far: RANSAC is solvable by opencv, with some proper set up, so I'm inclined towards that. 
WE can also use a program called Visual SFM http://ccwu.me/vsfm/ that I think can create 3d reconstructions and be used to test our stuff, at least as a baseline. Need to sit down and diagram this out super hard as well, though... Do that when we have time soon.

**Comment by dmurds** [comment]
Was just looking at CONOPS, doesn't look like this is technically still necessary

**Comment by benjaminwinger** [comment]
True, though I think it's worth keeping around and working on as a long-term part of the project since they'll probably re-introduce volume calculation in the future (though it makes me wonder if the only reason they had it last time was because of the company sponsoring the software).
